jsapp/js/components/processing/analysis/list/analysisQuestionRow.component.tsx

```diff
@@ -55,7 +57,20 @@ export default function AnalysisQuestionRow(props: AnalysisQuestionRowProps) {
     return null;
   }
 
-  const isDragDisabled = analysisQuestions.state.isPending;
+  // Reordering analysis questions requires `manage_asset` permission.
+  const hasManagePermissions = (() => {
+    const asset = assetStore.getAsset(singleProcessingStore.currentAssetUid);
+    return userCan('manage_asset', asset);
+  })();
```

Comment:
Not sure if this is a bug, an upstream issue, or just me doing something I shouldn't: if you remove the `manage_asset` permission from a user, the questions still show up as being editable in the UI.

![some user in admin](https://github.com/kobotoolbox/kpi/assets/7819986/8649a9d6-4712-4dc1-850b-a1bf73464470)

![what some user sees in the UI](https://github.com/kobotoolbox/kpi/assets/7819986/7886d59a-bb96-4328-b5e4-11ef7faf7024)

But then they get a 403 when submitting a change:
![verboten](https://github.com/kobotoolbox/kpi/assets/7819986/24bec76c-ea2a-41f6-aa42-963fc554b7d0)

It looks like this is because `userCan()` assumes the asset owner has full permissions:
```ts
  // If you own the asset, you can do everything with it
  if (asset.owner__username === currentUsername) {
    return true;
  }
```

Reply:
Thanks, @LMNTL: this is good to flag, but in this case, it's the third option :wink: 
We actually [grant all model-level permissions](https://github.com/kobotoolbox/kpi/blob/92b84b5c24f6c4ee78508e1eb672e88df89c2cbc/kpi/signals.py#L22-L37) to all users when they are first created and then manage permissions at the object level. We discourage people from mucking around in the Django admin permission settings and emphasize that changes there are unsupported. We also broadly assume that the owner has full control of the objects they own.